### PR TITLE
zh-trans: add docs/reference/access-authn-authz/abac.md

### PR DESCRIPTION
**Pull request by SataQiu**
Fix: k8smeetup/website-tasks#2106

### CODE REVIEW

**Comment by markthink** [comment]
@SataQiu  注释部分有问题：https://deploy-preview-15785--k8s-v1-14.netlify.com/zh/docs/reference/access-authn-authz/abac/

**Comment by SataQiu** [comment]
@markthink 已更新！

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+<!--
+To enable `ABAC` mode, specify `--authorization-policy-file=SOME_FILENAME` and `--authorization-mode=ABAC` on startup.
+-->
+要启用 `ABAC` 模式，请在启动时指定 `--authorization-policy-file=SOME_FILENAME` 和 `--authorization-mode=ABAC` 。
+
+<!--
+The file format is [one JSON object per line](http://jsonlines.org/).  There should be no enclosing list or map, just one map per line.
+-->
+文件格式是[每行一个 JSON 对象](http://jsonlines.org/)。不应该有半开的 list 或 map，每一行只有一个 map 。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
文件格式是[每行一个 JSON 对象](http://jsonlines.org/)。不允许存在内嵌的 list 或 map，每一行只能有一个 map 。
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        - Wildcard: 
+          - `*` matches all non-resource requests.
+          - `/foo/*` matches all subpaths of `/foo/`.
+    - `readonly`, type boolean, when true, means that the Resource-matching policy only applies to get, list, and watch operations, Non-resource-matching policy only applies to get operation.
+
+-->
+每一行都是一个“策略对象”，其中每个这样的对象都是具有以下属性的 map：
+
+- 版本属性：
+    - `apiVersion`， 字符串类型，有效值是 "abac.authorization.kubernetes.io/v1beta1"。 允许版本化和转换策略格式。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
    - `apiVersion`， 字符串类型，有效值是 "abac.authorization.kubernetes.io/v1beta1"。 用来支持策略格式的版本化和转换。
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+    - `kind`，字符串类型，有效值是 "Policy"。 允许版本化和转换策略格式。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
    - `kind`，字符串类型，有效值是 "Policy"。 用来支持策略格式的版本化和转换。
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+  - `spec` 属性设置为具有以下属性的 map：
+    - 用户匹配属性：
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
    - 主体匹配属性：
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+        - 通配符：
+          - `*` 匹配所有非资源请求。
+          - `/foo/*` 匹配 `/foo/` 的所有子路径。
+    - `readonly`，布尔类型， 表示资源匹配策略仅适用于 get，list 和 watch 操作，非资源匹配策略仅适用于 get 操作。
+
+{{< note >}}
+<!--
+An unset property is the same as a property set to the zero value for its type(e.g. empty string, 0, false). However, unset should be preferred for readability. In the future, policies may be expressed in a JSON format, and managed via aREST interface.
+-->
+未设置的属性与为其类型设置为零值的属性相同（例如，空字符串，0，false）。为了可读性，应该首选不重复设置保持默认值的内容。将来，策略可以用 JSON 格式表示，并通过一个 REST 接口进行管理。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
未设置的属性与为属性设置为对应类型之零值（例如，空字符串，0，false）的效果相同。为了可读性，应尽量采用不设置的方式。将来，策略可能会用 JSON 格式表示，并通过 REST 接口管理。
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 6]
```diff
@@ -0,0 +1,293 @@
...
+
+<!--
+A request has attributes which correspond to the properties of a policy object.
+-->
+请求具有与策略对象的属性相对应的属性。
+
+<!--
+When a request is received, the attributes are determined.  Unknown attributesare set to the zero value of its type (e.g. empty string, 0, false).
+-->
+当收到请求时，确定属性。未知属性设置为其类型的零值（例如空字符串，0，false）。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
当 kubernetes 收到请求时，确定各属性的取值。未知属性会被设置为其对应类型的零值（例如空字符串，0，false）。
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 7]
```diff
@@ -0,0 +1,293 @@
...
+
+<!--
+A property set to `"*"` will match any value of the corresponding attribute.
+-->
+设置为 `"*"` 的属性将匹配相应属性的任何值。
+
+<!--
+The tuple of attributes is checked for a match against every policy in thepolicy file. If at least one line matches the request attributes, then the request is authorized (but may fail later validation).
+-->
+检查属性元组是否与策略文件中的每个策略匹配。如果至少有一行匹配请求属性，则请求被授权（但稍后可能会验证失败）。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
属性元组与策略文件中的各个策略逐一比较，以寻找匹配项。如果至少有一行匹配请求属性，则请求被授权（但稍后可能会验证失败）。
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 8]
```diff
@@ -0,0 +1,293 @@
...
+
+<!--
+    kubectl --v=8 version
+-->
+kubectl --v=8 version
+
+<!--
+## Examples
+-->
+## 示例
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
## 示例 {#examples}
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 9]
```diff
@@ -0,0 +1,293 @@
...
+
+<!--
+## A quick note on service accounts
+-->
+## 关于服务帐户的快速说明
+
+<!--
+Every service account has a corresponding ABAC username, and that service account's user name is generated according to the naming convention:
+-->
+每个服务帐户都有一个相应的 ABAC 用户名，该服务帐户的用户名是根据命名约定生成的：
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
每个服务帐户都有一个相应的 ABAC 用户名，该服务帐户的用户名是根据下面的命名规则生成的：
```

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 10]
```diff
@@ -0,0 +1,293 @@
...
+例如，如果您想使用 ABAC 将默认服务帐户（在 `kube-system` 命名空间中）进行 API 完全授权，您可以将此行添加到您的策略文件中：
+
+```json
+{"apiVersion":"abac.authorization.kubernetes.io/v1beta1","kind":"Policy","spec":{"user":"system:serviceaccount:kube-system:default","namespace":"*","resource":"*","apiGroup":"*"}}
+```
+
+<!--
+The apiserver will need to be restarted to pickup the new policy lines.
+-->
+需要重新启动 apiserver 以生效新的策略行。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md]:
```suggestion
需要重新启动 apiserver 以使新的策略行生效。
```

**Comment by SataQiu** [comment]
Thank you so much! @tengqm

**Pull request merged**